apps/framework-cli/src/cli/logger.rs

```diff
@@ -151,6 +164,44 @@ pub fn setup_logging(settings: &LoggerSettings) -> Result<(), fern::InitError> {
         format_config.chain(fern::log_file(&settings.log_file)?)
     };
 
+    let output_config = match &settings.export_to {
+        None => output_config,
+        Some(otel_endpoint) => {
+            let https = hyper_tls_0_5::HttpsConnector::new();
+            let client = hyper_0_14::Client::builder().build::<_, hyper_0_14::Body>(https);
+
+            let otel_exporter = opentelemetry_otlp::new_exporter()
+                .http()
+                .with_endpoint(otel_endpoint.clone())
+                .with_protocol(Protocol::HttpJson)
+                .with_http_client(HyperClient::new_with_timeout(
+                    client,
+                    Duration::from_millis(100),
+                ))
+                .with_timeout(Duration::from_millis(10000))
+                .build_log_exporter()
+                .unwrap();
+
+            let logger_provider =
+                LoggerProvider::builder()
+                    .with_config(Config::default().with_resource(Resource::new(vec![
+                        KeyValue::new(SERVICE_NAME, "moose-cli"),
+                    ])))
+                    .with_batch_exporter(otel_exporter, opentelemetry_sdk::runtime::Tokio)
```

Comment:
`with_simple_exporter` gave me a dead lock when there's concurrency.
i had to do a thread dump inside a linux container to understand what went wrong.